### PR TITLE
require-imports-alphabetized: Adds new rule

### LINKED ISSUE #2049: New Rule: Alphabetize Imports
It might be nice to have a rule that requires imports to be alphabetized. Even more useful would be a rule that forces you to separate local imports from external imports with a newline.

### PR DESCRIPTION
**Pull request by rheh**
Adds new rule to enforce any import statement is alphabetically ordered.

Closes gh-2091
Fixes #2049 - issue

### CODE REVIEW

**Comment by hzoo** [comment]
We should do an autofix helper for sorting (talking to the core team)

File: lib/rules/require-imports-alphabetized.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    check: function(file, errors) {
+        var previous;
+        var current;
+
+        var lines = file.getLines();
+
+        for (var i = 0, l = lines.length; i < l; i++) {
+
+            if (lines[i].match(/import.*$/)) {
```
Comment (markelog) [inline lib/rules/require-imports-alphabetized.js]: Would you mind using ast instead here? Such checks are very unstable

**Comment by rheh** [comment]
@markelog sure,  I'll give it a go!

**Comment by rheh** [comment]
This pull request is now ready for review.  Thanks.

**Comment by markelog** [comment]
Awesome, thank you!

**Pull request closed without merging**